### ISSUE TITLE
Corneal pterygium

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7532]
**Preferred term label:**
Corneal pterygium

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:245660   PMID: 12915477  6/6

**Your nano-attribution (ORCID)**

**Comment by Sukramg** [comment]
Preferred term label:
Corneal pterygium

Synonyms
Pterygium
wing-shaped conjunctiva

Definition (free text, please give PubMed ID)
Pterygium (also known as surfer’s eye) is an ocular surface disease characterized mainly by a wing-shaped growth of limbal and conjunctival tissue over the adjacent cornea.
PMID: 34104871

Parent term (use hpo.jax.org/app)
Abnormal conjunctiva morphology HP:0000502
https://hpo.jax.org/app/browse/term/HP:0000502

Diseases characterized by this term ? (e.g. Orphanet or OMIM number)
OMIM:245660 PMID: 12915477 6/6

OMIM:178000 -> https://hpo.jax.org/app/browse/term/HP:0000502

Your nano-attribution (ORCID)
0000-0002-6601-4486

**Comment by pnrobinson** [comment]
added, thanks @Sukramg
